pkg/services/featuremgmt/goff_provider.go

```diff
@@ -0,0 +1,20 @@
+package featuremgmt
+
+import (
+	"net/http"
+	"time"
+
+	gofeatureflag "github.com/open-feature/go-sdk-contrib/providers/go-feature-flag/pkg"
+	"github.com/open-feature/go-sdk/openfeature"
+)
+
+func newGOFFProvider(url string) (openfeature.FeatureProvider, error) {
+	options := gofeatureflag.ProviderOptions{
+		Endpoint: url,
+		HTTPClient: &http.Client{
+			Timeout: 10 * time.Second,
+		},
+	}
+	provider, err := gofeatureflag.NewProvider(options)
```

Comment:
maybe we want a context (then we need to pass one in to `newGOFFProvider` etc...

```suggestion
	provider, err := gofeatureflag.NewProviderWithContext(ctx, options)
```

Reply:
Actually let's not, currently `ctx` is unused in `NewProviderWithContext`, so not worth the complication.